src/main/java/core/mate/academy/service/ExcavatorProducer.java

```diff
@@ -0,0 +1,11 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Excavator;
+import java.util.List;
+
+public class ExcavatorProducer implements MachineProducer<Excavator> {

```

Comment:
The ExcavatorProducer class should not begin with an empty line according to the checklist.